stdlib/testing/influxql/elapsed_test.flux

```diff
@@ -0,0 +1,703 @@
+,,1,1970-01-05T13:00:00Z,m,1,3600000000000
+,,1,1970-01-05T14:00:00Z,m,1,3600000000000
+,,1,1970-01-05T15:00:00Z,m,1,3600000000000
+,,1,1970-01-05T16:00:00Z,m,1,3600000000000
+,,1,1970-01-05T17:00:00Z,m,1,3600000000000
+,,1,1970-01-05T18:00:00Z,m,1,3600000000000
+,,1,1970-01-05T19:00:00Z,m,1,3600000000000
+,,1,1970-01-05T20:00:00Z,m,1,3600000000000
+,,1,1970-01-05T21:00:00Z,m,1,3600000000000
+,,1,1970-01-05T22:00:00Z,m,1,3600000000000
+,,1,1970-01-05T23:00:00Z,m,1,3600000000000
+,,1,1970-01-06T00:00:00Z,m,1,3600000000000
+,,1,1970-01-06T01:00:00Z,m,1,3600000000000
+,,1,1970-01-06T02:00:00Z,m,1,3600000000000
+,,1,1970-01-06T03:00:00Z,m,1,3600000000000
+,,1,1970-01-06T04:00:00Z,m,1,3600000000000
+,,1,1970-01-06T05:00:00Z,m,1,3600000000000
+,,1,1970-01-06T06:00:00Z,m,1,3600000000000
+,,1,1970-01-06T07:00:00Z,m,1,3600000000000
+,,1,1970-01-06T08:00:00Z,m,1,3600000000000
+,,1,1970-01-06T09:00:00Z,m,1,3600000000000
+,,1,1970-01-06T10:00:00Z,m,1,3600000000000
+,,1,1970-01-06T11:00:00Z,m,1,3600000000000
+,,1,1970-01-06T12:00:00Z,m,1,3600000000000
+,,1,1970-01-06T13:00:00Z,m,1,3600000000000
+,,1,1970-01-06T14:00:00Z,m,1,3600000000000
+,,1,1970-01-06T15:00:00Z,m,1,3600000000000
+,,1,1970-01-06T16:00:00Z,m,1,3600000000000
+,,1,1970-01-06T17:00:00Z,m,1,3600000000000
+,,1,1970-01-06T18:00:00Z,m,1,3600000000000
+,,1,1970-01-06T19:00:00Z,m,1,3600000000000
+,,1,1970-01-06T20:00:00Z,m,1,3600000000000
+,,1,1970-01-06T21:00:00Z,m,1,3600000000000
+,,1,1970-01-06T22:00:00Z,m,1,3600000000000
+,,1,1970-01-06T23:00:00Z,m,1,3600000000000
+,,1,1970-01-07T00:00:00Z,m,1,3600000000000
+,,1,1970-01-07T01:00:00Z,m,1,3600000000000
+,,1,1970-01-07T02:00:00Z,m,1,3600000000000
+,,1,1970-01-07T03:00:00Z,m,1,3600000000000
+,,1,1970-01-07T04:00:00Z,m,1,3600000000000
+,,1,1970-01-07T05:00:00Z,m,1,3600000000000
+,,1,1970-01-07T06:00:00Z,m,1,3600000000000
+,,1,1970-01-07T07:00:00Z,m,1,3600000000000
+,,1,1970-01-07T08:00:00Z,m,1,3600000000000
+,,1,1970-01-07T09:00:00Z,m,1,3600000000000
+,,1,1970-01-07T10:00:00Z,m,1,3600000000000
+,,1,1970-01-07T11:00:00Z,m,1,3600000000000
+,,1,1970-01-07T12:00:00Z,m,1,3600000000000
+,,1,1970-01-07T13:00:00Z,m,1,3600000000000
+,,1,1970-01-07T14:00:00Z,m,1,3600000000000
+,,1,1970-01-07T15:00:00Z,m,1,3600000000000
+,,1,1970-01-07T16:00:00Z,m,1,3600000000000
+,,1,1970-01-07T17:00:00Z,m,1,3600000000000
+,,1,1970-01-07T18:00:00Z,m,1,3600000000000
+,,1,1970-01-07T19:00:00Z,m,1,3600000000000
+,,1,1970-01-07T20:00:00Z,m,1,3600000000000
+,,1,1970-01-07T21:00:00Z,m,1,3600000000000
+,,1,1970-01-07T22:00:00Z,m,1,3600000000000
+,,1,1970-01-07T23:00:00Z,m,1,3600000000000
+"
+
+// SELECT elapsed(f) FROM m GROUP BY *
+t_elapsed = (tables=<-) => tables
+	|> range(start: influxql.minTime, stop: influxql.maxTime)
+	|> filter(fn: (r) => r._measurement == "m" and r._field == "f")
+	|> elapsed(unit: 1ns)
+	|> map(fn: (r) => ({r with elapsed: float(v: r.elapsed)}))
+	|> drop(columns: ["_start", "_stop", "_field", "_value"])
+	|> rename(columns: {_time: "time"})
```

Comment:
There was some discussion [here](https://github.com/influxdata/influxdb/issues/10733) that some columns might need to be renamed. Since `elapsed()`  produces a column that is already called `elapsed` by default, I didn't find that necessary here.